src/main/java/config/WebConfig.java

```diff
@@ -0,0 +1,23 @@
+package config;
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.web.servlet.config.annotation.EnableWebMvc;
+import org.springframework.web.servlet.view.InternalResourceViewResolver;
+import org.springframework.web.servlet.view.JstlView;
+
+@Configuration
+@EnableWebMvc
+@ComponentScan("controller")
+public class WebConfig {
+
+    @Bean
+    public InternalResourceViewResolver resolver() {
```

Comment:
it method returns bean, so lets name it getResolver() for example. Method name should start from verb

Reply:
Done